### PR TITLE
Specs2 produced junit xml report as additional test output

### PR DESCRIPTION
**Pull request by hmemcpy**
This is needed to use the report with external tools, e.g. comparing against junit xml reports produced by other runners (e.g. Surefire)

@ittaiz please review

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: specs2/BUILD [hunk 1]
```diff
@@ -1,13 +1,14 @@
+load("//scala:scala_import.bzl", "scala_import")
+
 package(default_visibility = ["//visibility:public"])
-java_import(
+scala_import(
```
Comment (ittaiz) [inline specs2/BUILD]: Why is this change needed?

**Comment by hmemcpy** [comment]
Hmm, I saw this, but I was sure this was just for scalatest?

File: specs2/BUILD (hunk 1 above)
Comment (hmemcpy) [reply specs2/BUILD]: I think it's a leftover from our change with the jars, I'll undo and see what happens.

**Comment by ittaiz** [comment]
build fails...

**Comment by hmemcpy** [comment]
Maybe it just doesn't like me? :)
I'll check...

File: specs2/BUILD (hunk 1 above)
Comment (hmemcpy) [reply specs2/BUILD]: Ok, I reverted it back to java_import, still works!

**Comment by hmemcpy** [comment]
After further discussion with @ittaiz, I'm closing this PR in favor of a local cherry-pick for this. We need this temporarily to verify the migration was done correctly, and there's no reason to keep this in rules_scala.
